torch/_torch_docs.py

```diff
@@ -4018,6 +4018,72 @@ def parse_kwargs(desc):
             [ 1,  2,  2,  0]])
 """)
 
+add_docstr(torch.sparse_coo_tensor,
+           r"""
+sparse_coo_tensor(indices, values, size=None, dtype=None, device=None, requires_grad=False) -> Tensor
+
+Constructs a sparse_coo_tensor with non-zero elements at the given :attr:`indices` with the given
+:attr:`values`.
+
+Args:
+    indices (array_like): Initial data for the tensor. Can be a list, tuple,
+        NumPy ``ndarray``, scalar, and other types. Will be cast to a :class:`torch.LongTensor`
```

Comment:
@SsnL @zou3519 Is this following the convention for how we document these kinds of arguments?

Reply:
Just for reference, the first two sentences are taken directly from the doc for `torch.tensor`. The latter part was added to clarify what type is expected and that indices are _coordinates_. Let me know if I need to change anything.